postman/collection-dir/adyen_uk/Flow Testcases/QuickStart/Merchant Account - Create/request.json

```diff
@@ -79,6 +83,14 @@
       "metadata": {
         "city": "NY",
         "unit": "245"
+      },
+      "routing_algorithm": {
```

Comment:
Adding `routing_algorithm` will cause the collection to fail. Kindly remove it. Instead, add it in `Payment Create` step.

Reply:
Why would the collection fail? I have tested this locally, adding this here means there's no need to pass this individually in txn requests
![image](https://github.com/juspay/hyperswitch/assets/46213975/27193862-9184-4ba4-9bfc-0e2cfe6f916e)

Reply:
You can refer this PR for that: https://github.com/juspay/hyperswitch/pull/2032

Reply:
I ran the collection locally, 14 assertions failed. 4 failures is related to manual capture being not allowed from the connector, and the rest 10 failures is related to refunds not being processed (They're going to `pending` state instead of `succeeded`. Same is happening with `Stripe` as well which can be seen below in the CI checks).

Reply:
I have moved routing_algorithm to payment and payout create requests.

![image](https://github.com/juspay/hyperswitch/assets/46213975/1a9167f3-0e53-4822-be58-b73ee7f148da)